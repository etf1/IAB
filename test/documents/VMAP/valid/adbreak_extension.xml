<!-- AdBreaks with different Extensions values -->
<vmap:VMAP version='1.0' xmlns:vmap='http://www.iab.net/vmap-1.0'>
    <vmap:AdBreak breakType='linear' timeOffset="start" >
        <vmap:Extensions>
            <vmap:Extension type="http://www.google.fr">
                <root><tag>1</tag></root>
            </vmap:Extension>
        </vmap:Extensions>
    </vmap:AdBreak>
    <vmap:AdBreak breakType='linear' timeOffset="start" >
        <vmap:Extensions>
            <vmap:Extension type="http://www.google.fr">
                <![CDATA[Test]]>
            </vmap:Extension>
        </vmap:Extensions>
    </vmap:AdBreak>
    <vmap:AdBreak breakType='linear' timeOffset="start" >
        <vmap:Extensions>
            <vmap:Extension type="http://www.google.fr">1</vmap:Extension>
        </vmap:Extensions>
    </vmap:AdBreak>
    <vmap:AdBreak breakType='linear' timeOffset="start" >
        <vmap:Extensions>
            <vmap:Extension type="empty"></vmap:Extension>
        </vmap:Extensions>
    </vmap:AdBreak>
    <vmap:AdBreak breakType='linear' timeOffset="start" >
        <vmap:Extensions>
            <vmap:Extension type="empty"></vmap:Extension>
            <vmap:Extension type="url">http://www.google.fr</vmap:Extension>
            <vmap:Extension type="http://www.google.fr">
                <![CDATA[Test]]>
            </vmap:Extension>
            <vmap:Extension type="http://www.google.fr">
                <xml><a>1</a></xml>
            </vmap:Extension>
        </vmap:Extensions>
    </vmap:AdBreak>
</vmap:VMAP>
