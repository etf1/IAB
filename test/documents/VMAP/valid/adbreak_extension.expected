{
  "version": "1.0",
  "breaks": [
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "extensions": [
        {
          "extensionType": "http://www.google.fr",
          "value": {
            "type": "http://www.google.fr",
            "root": [
              {
                "tag": [
                  {
                    "$t": "1"
                  }
                ]
              }
            ]
          }
        }
      ]
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "extensions": [
        {
          "extensionType": "http://www.google.fr",
          "value": "Test"
        }
      ]
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "extensions": [
        {
          "extensionType": "http://www.google.fr",
          "value": "1"
        }
      ]
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "extensions": [
        {
          "extensionType": "empty",
          "value": ""
        }
      ]
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "extensions": [
        {
          "extensionType": "empty",
          "value": ""
        },
        {
          "extensionType": "url",
          "value": "http://www.google.fr"
        },
        {
          "extensionType": "http://www.google.fr",
          "value": "Test"
        },
        {
          "extensionType": "http://www.google.fr",
          "value": {
            "type": "http://www.google.fr",
            "xml": [
              {
                "a": [
                  {
                    "$t": "1"
                  }
                ]
              }
            ]
          }
        }
      ]
    }
  ]
}
